portfolio/src/main/webapp/load-index.js

```diff
@@ -1,6 +1,18 @@
 // Set the nametag's HTML to be that fetched from the server
 async function getNametagText() {
   const response = await fetch('/nametag-text');
-  const greeting = await response.text();
-  document.getElementById('nametag').innerHTML = greeting;
+  const greetings = await response.json();
+
+  // Properly format and place the fetched greetings within the nametag
+  const nametag = document.getElementById('nametag'); 
+  nametag.innerHTML = formatComments(greetings);
+}
+
+// Format a list of comments such that their is a newline between each
+function formatComments (comments) {
```

Comment:
nit: remove the space, and fix typo in comments

```javascript
// Format a list of comments such that *there* is a newline between each
function formatComments(comments) {

}
```

Reply:
my bad! fixed